src/test/java/karate/KarateRunner.java

```diff
@@ -0,0 +1,40 @@
+package karate;
+
+import com.intuit.karate.Results;
+import com.intuit.karate.Runner;
+import net.masterthought.cucumber.Configuration;
+import net.masterthought.cucumber.ReportBuilder;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class KarateRunner {
+
+    /**
+     * This method generates the .json which will then be used to build the cucumber report
+     *
+     * @param karateOutputPath (In this path the json are kept)
+     */
+    private static void generateReport(String karateOutputPath) {
+        Collection<File> jsonFiles = FileUtils.listFiles(new File(karateOutputPath), new String[]{"json"}, true);
+        List<String> jsonPaths = new ArrayList<>(jsonFiles.size());
+        jsonFiles.forEach(file -> jsonPaths.add(file.getAbsolutePath()));
+        Configuration config = new Configuration(new File("target"), "name-api");
+        ReportBuilder reportBuilder = new ReportBuilder(jsonPaths, config);
+        reportBuilder.generateReports();
+    }
+
+    @Test
+    public void testParallel() {
+        String karateOutputPath = "target/surefire-reports";
+        Results results = Runner.parallel(getClass(), 5, karateOutputPath);
+        generateReport(results.getReportDir());
+        Assertions.assertEquals(0, results.getFailCount());
+    }
+
+}
```

Comment:
It may or may not go in that location, it depends on the archetype and the type of project to implement, it would be a task for the QA to locate it in the right place.